test/support/assertions.rb

```diff
@@ -57,6 +57,21 @@ def assert_no_line_text(text)
       })
     end
 
+    def assert_debuggee_out exp
```

Comment:
I think it should be consistent with `assert_line_text` (name and parameters)

Reply:
https://github.com/ruby/debug/pull/365

Reply:
Just now it is not renamed yet, so it should be renamed after #308.
Or merge this pull request after #308?

Reply:
Either is fine, but if I had to say, I prefer the former. To make it consistent with #365, we may need to change `assert_debuggee_out`, too.

Reply:
Former option means introduce current consistent name (maybe `assert_debugee_line_text` or similar) and rename it with #365. Could you try it?

Reply:
I renamed it to "assert_debuggee_line_text".